pkgs/applications/networking/instant-messengers/patchwork/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, writeScriptBin, path, fetchurl, fetchFromGitHub
+, nodejs-8_x, yarn, python, electron }:
+
+with builtins;
+
+let
+  version = "3.8.10";
+
+  nodejs = nodejs-8_x;
+
+  yarn2nix = (import (fetchFromGitHub {
+    owner = "moretea";
+    repo = "yarn2nix";
+    rev = "d6e05a521bd92b2647bb7e853363d234f21b2cfd";
+    sha256 = "1nvpii9p41vrb6zvr8rqcvmycrl6lnzzaif85qj1aavizncgb4wy";
+  }) {
```

Comment:
last time I checked import from derivation was not allowed on Hydra, we would have import yarn2nix into nixpkgs first

Reply:
Just need to find a good location for it... I'll start working on a separate PR. Just hope @moretea is OK with it.

Reply:
I'm quite sure he's OK with it. The only reason not to include it is that a new yarn2nix nix emerged that is even more powerful: https://github.com/Profpatsch/yarn2nix